service-front/module/Application/tests/View/Helper/MoneyFormatTest.php

```diff
@@ -18,7 +18,7 @@ public function testInvokeFloor():void
 
     public function testInvokeCeil():void
     {
-        $amount= 50.55;
+        $amount= 50.56;
```

Comment:
This seems an odd change. Do we know why it happened?

Reply:
Neil, I made the change as the test was breaking with the error

Reply:
@SeemaMenon Sorry, I mean I assume the test was once passing with 50.55. So did something change so that the test needs updating? Or was the test always failing and needing 50.56?